.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I believe this git ignore is missing more entries (IDE files, system folder files, etc.)